metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -64,7 +65,16 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
 )
 @click.option(
     "--experiment-name",
-    "experiment_name",
+    "experiment",
+    default=None,
+    help="Deprecated. Please use --experiment option."
+    "Default of None uses KFP 'default' experiment",
+    show_default=True,
```

Comment:
This could be combined and mark the old option as deprecated.

Reply:
as we discussed offline, I also was reading this exact article this morning. I was surprised there is no native way for deprecation. Given that this is literally will live for a day or two until we update the CICD, it is way too much extra engineering that probably doesn't worth it...